### ISSUE TITLE
About Layer Weights αk

### ISSUE THREAD

**Issue by lbdbj** [gusye1234/LightGCN-PyTorch#33]
大佬您好，您的论文中有提到将每层embedding的权重设置为1/(K+1)，但在代码中并没有体现，请问这是怎么回事

**Comment by zbt78** [comment]
在model.py的 107行
`light_out = torch.mean(embs, dim=1)`
作者好像是对这三层求平均值了.
